examples/docker-gen.cfg

```diff
@@ -8,3 +8,11 @@ notifycmd = "/etc/init.d/nginx reload"
 template = "/etc/logrotate.conf.tmpl"
 dest = "/etc/logrotate.d/docker"
 watch = true
+
+[[config]]
+Template = "/etc/docker-gen/templates/nginx.tmpl"
+Dest = "/etc/nginx/conf.d/default.conf"
+watch = true
+[config.NotifyContainers]
+nginx = 1  # 1 is a signal number to be sent; here SIGINT
```

Comment:
Honestly I'm used to kill -1 rather than kill -SIGINT so 1 seemed more logical to me :)